wfms/camunda/v1/WfMSPlugin.java

```diff
@@ -0,0 +1,77 @@
+package cloud.benchflow.plugins.camunda.v1;
```

Comment:
We should remove the version from the package, because if we have version ranges like `v1-v4`, this is not a valid package name.